app/models/user.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable func-names */
+/* eslint-disable no-unused-vars */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
```

Comment:
warning, you use singular here but in migration use plural.
For convention we use plural for tables.